flow/project.py

```diff
@@ -1007,75 +1012,95 @@ def eligible(self, job, ignore_conditions=IgnoreConditions.NONE):
         A FlowGroup is eligible for execution if at least one of its
```

Comment:
```suggestion
        A :class:`~.FlowGroup` is eligible for execution if at least one of its
```
I saw this change in line 971 of the diff (`project.py`). I think this should be reflected everywhere else?